packages/next/src/client/components/react-dev-overlay/internal/container/RuntimeError/index.tsx

```diff
@@ -43,7 +43,14 @@ const RuntimeError: React.FC<RuntimeErrorProps> = function RuntimeError({
     [all, allLeadingFrames]
   )
   const allCallStackFrames = React.useMemo<OriginalStackFrame[]>(
-    () => error.frames.slice(firstFirstPartyFrameIndex + 1),
+    () =>
+      error.frames
+        .slice(firstFirstPartyFrameIndex + 1)
+        .filter(
+          (f) =>
+            f.sourceStackFrame.file !== '<anonymous>' &&
+            !f.sourceStackFrame.file?.startsWith('node:internal')
+        ),
```

Comment:
Any kind of manipulation of the frames (either `filter`ing or `map`ing, like hiding useless frames or stripping `webpack://` from the file name, etc.) could be extracted to a single file for better visibility.)

Reply:
We only have `webpack://` as start traces when you run `pnpm next ...`, it could be other cases. We need a test for both app router and paages router